src/plots/cartesian/graph_interact.js

```diff
@@ -18,7 +20,13 @@ var makeDragBox = require('./dragbox').makeDragBox;
 module.exports = function initInteractions(gd) {
     var fullLayout = gd._fullLayout;
 
-    if((!fullLayout._has('cartesian') && !fullLayout._has('gl2d')) || gd._context.staticPlot) return;
+    if(gd._context.staticPlot) {
+        // this sweeps up more than just cartesian drag elements...
+        d3.select(gd).selectAll('.drag').remove();
```

Comment:
Far as I could tell this is the only change needed so `config` can be altered with a regular `Plotly.plot` call - everything else I tried just seemed to work. If we make more of a `supplyDefaults` type framework for config at some point we may be able to minimize the update pathway there too, like modebar-only changes... but it's probably not a very common use case anyhow.

Reply:
What about other svg-based subplot types? For example, I'm thinking we could add a similar `.remove()` statement for polar [here](https://github.com/plotly/plotly.js/blob/ef2bff1904b50fa6fd01655ac37a588866a0e330/src/plots/polar/polar.js#L534)

Reply:
we could... though this one kills the polar draggers too, hence my comment. I originally put the `remove` inside the one `if((!fullLayout._has('cartesian') && !fullLayout._has('gl2d')) || gd._context.staticPlot)` block, and it broke all polar drag and hover effects.

You were saying something about overly broad `selectAll` statements the other day 😏 It wasn't obvious to me how to make this more specific though...

Reply:
> though this one kills the polar draggers too,

Oh nice. Is this tested? If not, would you mind :lock: ing this down?

Reply:
🔒 https://github.com/plotly/plotly.js/pull/2341/commits/0548db3dd9e5eda6774d7e6f7f58a44c7453aef3